src/main/java/net/phptravels/gui/pages/HotelsSearchResultPage.java

```diff
@@ -0,0 +1,12 @@
+package net.phptravels.gui.pages;
+
+import org.openqa.selenium.WebDriver;
+
+import com.qaprosoft.carina.core.gui.AbstractPage;
+
+public class HotelsSearchResultPage extends AbstractPage {
+
+	public HotelsSearchResultPage(WebDriver driver){
+	    super(driver);
+	  }
```

Comment:
i think here we need to add method `isPageOpened`
And an element that youwill use as an indicator that page opened